quran/quran.com-frontend-next
# add ability to select/change translations

### Summary
This PR adds the ability to select/un-select multiple translations for the QuranReader.

### Test Plan

1. Open the Developer Utility.
2. `Show Translations` button should show.
3. Click on `Show Translations` button which should show a multiple `select` element that has the list of available translations.
4. Select the first 2 translations and they should appear below each other in the QuranReader.
5. Refresh the page, the 2 translations selected in step 4 should still show.
6. Open the Developer Utility again and click on `Show Translations` button again and the 2 translations selected in step 4 should still be selected.

### Screenshots

| Before | After |
| ------ | ------ |
| <img width="1440" alt="Before" src="https://user-images.githubusercontent.com/15169499/126036213-475eca74-54b7-4a92-bae6-3f12bc77119d.png"> | <img width="1439" alt="After 1" src="https://user-images.githubusercontent.com/15169499/126036207-9d33f573-0129-492f-99f4-5f41db2379b7.png"> |
|  | <img width="1440" alt="After 2" src="https://user-images.githubusercontent.com/15169499/126036210-4460d64a-e4eb-4c8a-a963-1a49ab85b87d.png"> |
|  | <img width="1440" alt="After 3" src="https://user-images.githubusercontent.com/15169499/126036211-91174659-e51b-46fe-9be6-adb6147a9a4d.png"> |
|  | <img width="1440" alt="After 4" src="https://user-images.githubusercontent.com/15169499/126036212-522a7fcb-0425-4a1c-aa49-18ed30753c01.png"> |
